code/addons/actions/README.md

```diff
@@ -8,13 +8,13 @@ Storybook Addon Actions can be used to display data received by event handlers i
 
 ## Installation
 
-Actions is part of [essentials](https://storybook.js.org/docs/react/essentials/introduction) and so is installed in all new Storybooks by default. If you need to add it to your Storybook, you can run:
+Actions is part of [essentials](https://storybook.js.org/docs/react/essentials) and so is installed in all new Storybooks by default. If you need to add it to your Storybook, you can run:
```

Comment:
@kylegach, one small item here. I suggest removing the framework in the links for our addons and providing with the installation process. Reasoning here: if the reader checks this but is working with Vue or has checked our Vue docs, they should not see a framework that's not his own. This leads him to have to go up the page and select another renderer just to see what wants to read about, otherwise it should automatically default to React. 
Just trying to keep it consistent, and I'll leave it to you to proceed.

Reply:
This PR corresponds with one in frontpage that completely removes the renderer from the URL. URLs containing a renderer will redirect to the the correct renderer-less URL with zero affect on whatever renderer is in their localStorage and used for the site.

Reply:
Additionally, I opted to _not_ update the various URLs in READMEs in this PR. While most renderers only have a few results, `react` has 150+ and I didn't want to make this one any bigger than it needed to be.